### ISSUE TITLE
Operations involving equality fail with java.time.Month in ClojureScript

### ISSUE THREAD

**Issue by SevereOverfl0w** [juxt/tick#30]
```
❯ clj -Sdeps '{:deps {tick {:mvn/version "RELEASE"} org.clojure/clojurescript {:mvn/version "1.10.516"}}}' -m cljs.main
Downloading: tick/tick/maven-metadata.xml from https://repo.clojars.org/
ClojureScript 1.10.516
cljs.user=> (require '[tick.alpha.api :as t])
nil
cljs.user=> (require 'time-literals.data-readers)
nil
cljs.user=> #{#time/month "MARCH" #time/month "APRIL"}
Execution error (Error) at (<cljs repl>:1).
x__$1.equals is not a function
```

**Comment by henryw374** [comment]
I'll aim to fix in js-joda: https://github.com/js-joda/js-joda/issues/299

**Comment by henryw374** [comment]
version with fix released in clojars [tick "0.4.12-alpha"]

**Comment by oakes** [comment]
This seems to be broken again in 0.4.13-alpha.

**Comment by henryw374** [comment]
Ah, thanks for pointing that out. cljc.java-time was not on the latest cljs.java-time. 

I have released 0.4..14-alpha now
